libzmq/test_package/conanfile.py

```diff
@@ -5,7 +5,7 @@
 
 class ZeroMQTest(ConanFile):
     settings = "os", "compiler", "arch", "build_type"
-    generators = "cmake_find_package"
```

Comment:
The CMake script in the `test_package` folder might look more complicated because it is more lines, but that's only because it was using a CMake script generated by Conan and was therefore _reliant_ on Conan. Using ZeroMQ outside of a Conan package would have been much more difficult. You'll notice here that we're not just using the standard "cmake" generator which means the CMake script used in `test_package` would work outside Conan as well.